server/tests/client_checks.py

```diff
@@ -0,0 +1,74 @@
+import logging
+import time
+from typing import Callable
+
+from channelfinder import ChannelFinderClient
+from testcontainers.compose import DockerCompose
+
+LOG: logging.Logger = logging.getLogger(__name__)
+
+ACTIVE_PROPERTY = {"name": "pvStatus", "owner": "admin", "value": "Active", "channels": []}
+INACTIVE_PROPERTY = {"name": "pvStatus", "owner": "admin", "value": "Inactive", "channels": []}
+MAX_WAIT_SECONDS = 180
+TIME_PERIOD_INCREMENT = 2
+
+
+def channel_match(channel0, channel1, properties_to_match: list[str]):
+    if channel0["name"] != channel1["name"]:
+        assert False, f"Channel names do not match: {channel0['name']} != {channel1['name']}"
+    if channel0["owner"] != channel1["owner"]:
+        assert False, f"Channel owners do not match: {channel0['owner']} != {channel1['owner']}"
```

Comment:
You can just do `assert condition, message`

Reply:
By jove, he's right!
```
>>> assert False, "foo"
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AssertionError: foo
>>> assert True, "bar"
>>>
```